uint16  counter
float64 nbUptime

float32 accX1
float32 accY1
float32 accZ1

float32 rateP1
float32 rateQ1
float32 rateR1

uint16  mainLoopCounter
float64 i7Clock
